analysis/combiners.py

```diff
@@ -358,9 +358,33 @@ def _to_dense(self,
             ]),
         )
 
+    def _merge_sparse(self, acc1, acc2):
+        if acc1 is None:
+            return acc2
+        if acc2 is None:
+            return acc1
+        return tuple(_merge_list(s, t) for s, t in zip(acc1, acc2))
```

Comment:
I don't remember this in detail. Was the sparse mode a tuple of lists?

Reply:
Yes, sparse mode is keeping raw values to aggregate (in lists).